python/lsst/meas/deblender/deblend.py

```diff
@@ -1175,7 +1175,7 @@ def skipParent(self, source, masks):
         ----------
         source: `lsst.afw.table.source.source.SourceRecord`
             The source to flag as skipped
-        masks: list of `lsst.afw.image.mask.mask.MaskX`
+        masks: list of `lsst.afw.image.MaskX`
```

Comment:
This maybe could be updated to just say Mask, as the Mask constructor will resolve to MaskX. It was designed so people would not need to know the details of the mask extension. However, as the concrete type is MaskX it would be appropriate to leave it as you have it as well.